lib/internal/test_runner/mock/mock_timers.js

```diff
@@ -48,7 +56,13 @@ function abortIt(signal) {
   return new AbortError(undefined, { cause: signal.reason });
 }
 
-const SUPPORTED_TIMERS = ['setTimeout', 'setInterval'];
+const SUPPORTED_TIMERS = ['setTimeout', 'setInterval', 'Date'];
```

Comment:
```suggestion
const SUPPORTED_APIS = ['setTimeout', 'setInterval', 'Date'];
```

Reply:
Will change this as a whole in another commit